src/services/ratingService.ts

```diff
@@ -28,6 +28,32 @@ class RatingService {
 		return ratings;
 	};
 
+	static getTopFiveAverageRatings = async (): Promise<object[]> => {
+		const ratingRepository: Repository<Rating> = getRepository(Rating);
+		const ratings: object[] = await ratingRepository
+			.createQueryBuilder('rating')
+			.innerJoin('rating.service', 'service')
+			.innerJoin('service.tutorInfo', 'tutorInfo')
+			.innerJoin('tutorInfo.user', 'user')
+			.groupBy('service.id')
+			.addGroupBy('user.id')
+			.select([
+				'service.id AS serviceId',
+				'service.name AS name',
+				'service.description AS description',
+				'service.tutorInfo',
+				'AVG(rating.rating) AS avgRating',
+				'user.id AS userId',
+				'user.firstName AS firstName',
+				'user.lastName AS lastName',
+			])
+			.orderBy('avgRating', 'DESC')
+			.limit(5)
```

Comment:
```suggestion
			.limit(x)
```

hvor x er hvad en der er kommet som parameter